  caused by patches.arch/ppc-ps3tree-spe-logo_logo-extern-in-header.diff

  CC      drivers/video/logo/logo_linux_clut224.o
  drivers/video/logo/logo_linux_clut224.c:548: error: logo_linux_clut224_clut causes a section type conflict
  drivers/video/logo/logo_linux_clut224.c:548: error: logo_linux_clut224_clut causes a section type conflict
  drivers/video/logo/logo_linux_clut224.c:11: error: logo_linux_clut224_data causes a section type conflict
  drivers/video/logo/logo_linux_clut224.c:11: error: logo_linux_clut224_data causes a section type conflict

---
 include/linux/linux_logo.h |   26 +++++++++++++-------------
 scripts/pnmtologo.c        |    2 +-
 2 files changed, 14 insertions(+), 14 deletions(-)

--- a/include/linux/linux_logo.h
+++ b/include/linux/linux_logo.h
@@ -32,19 +32,19 @@ struct linux_logo {
 	const unsigned char *data;
 };
 
-extern const struct linux_logo logo_linux_mono;
-extern const struct linux_logo logo_linux_vga16;
-extern const struct linux_logo logo_linux_clut224;
-extern const struct linux_logo logo_dec_clut224;
-extern const struct linux_logo logo_mac_clut224;
-extern const struct linux_logo logo_parisc_clut224;
-extern const struct linux_logo logo_sgi_clut224;
-extern const struct linux_logo logo_sun_clut224;
-extern const struct linux_logo logo_superh_mono;
-extern const struct linux_logo logo_superh_vga16;
-extern const struct linux_logo logo_superh_clut224;
-extern const struct linux_logo logo_m32r_clut224;
-extern const struct linux_logo logo_spe_clut224;
+extern struct linux_logo logo_linux_mono;
+extern struct linux_logo logo_linux_vga16;
+extern struct linux_logo logo_linux_clut224;
+extern struct linux_logo logo_dec_clut224;
+extern struct linux_logo logo_mac_clut224;
+extern struct linux_logo logo_parisc_clut224;
+extern struct linux_logo logo_sgi_clut224;
+extern struct linux_logo logo_sun_clut224;
+extern struct linux_logo logo_superh_mono;
+extern struct linux_logo logo_superh_vga16;
+extern struct linux_logo logo_superh_clut224;
+extern struct linux_logo logo_m32r_clut224;
+extern struct linux_logo logo_spe_clut224;
 
 extern const struct linux_logo *fb_find_logo(int depth);
 #if defined(CONFIG_LOGO) && defined(CONFIG_FB)
--- a/scripts/pnmtologo.c
+++ b/scripts/pnmtologo.c
@@ -244,7 +244,7 @@ static void write_header(void)
 static void write_footer(void)
 {
     fputs("\n};\n\n", out);
-    fprintf(out, "const struct linux_logo %s __initdata = {\n", logoname);
+    fprintf(out, "struct linux_logo %s __initdata = {\n", logoname);
     fprintf(out, "    .type\t= %s,\n", logo_types[logo_type]);
     fprintf(out, "    .width\t= %d,\n", logo_width);
     fprintf(out, "    .height\t= %d,\n", logo_height);
